guides/micronaut-cloud-database-aws/micronaut-cloud-database-aws.adoc

```diff
@@ -0,0 +1,177 @@
+external:micronaut-cloud-database-base/start.adoc[]
+* An https://aws.amazon.com/[AWS account] with:
+** An IAM user with enough permissions to create and manage DB instances in RDS.
+** The AWS CLI configured to use the IAM user above.
+* The https://dev.mysql.com/doc/refman/8.0/en/mysql.html[MySQL client CLI].
+
+common:aws.adoc[]
+
+== Creating a MySQL Database instance in Amazon Relational Database Service (Amazon RDS)
+
+We will create the database with the AWS CLI. See the https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/index.html[AWS CLI `rds` command] for more information.
+
+=== Create VPC, security group, subnets and subnet group (optional)
+
+To allow connections to the database from your local computer, we will create a VPC and a security group that allows access to the MySQL default port from your current public IP address.
+
+WARNING: Exposing a database port to the internet is a security risk. This should only be done for development purposes.
+
+We will also create 2 subnets in different availability zones, and a subnet group to associate them.
+
+[source,bash]
+----
+# VPC, internet gateway and route table
+export VPC_ID=$(aws ec2 create-vpc --cidr-block 10.0.0.0/16 | jq -r '.Vpc.VpcId')
```

Comment:
Should we add a link to jq?  https://stedolan.github.io/jq/

Reply:
it is not install by default in MacOS for example.

Reply:
Resolved via https://github.com/micronaut-projects/micronaut-guides/pull/798/commits/8c9e6c5566531e158b7e0fed33e826ad84011659